routers/api/v1/repo/tag.go

```diff
@@ -130,37 +130,22 @@ func DeleteTag(ctx *context.APIContext) {
 	//   required: true
 	// - name: tag
 	//   in: path
-	//   description: name of tag to delete
+	//   description: name of tag
 	//   type: string
 	//   required: true
 	// responses:
-	//   "204":
+	//   "200":
```

Comment:
You're not returning empty here

Reply:
No i return 200

Reply:
@zacheryph DeleteTag codeblog got moved ...

Reply:
Look at the line below

Reply:
haha right - good catch